fmn/rules/taskotron.py

```diff
@@ -5,7 +5,8 @@
 RELEASE_CRITICAL_TASKS = [
```

Comment:
That's up to the fmn devs, I guess. If they prepare some plumbing, we can certainly adjust the code.

However, I'd like to push this change now, and deal with config files later.